e2e/src/pages/app.po.ts

```diff
@@ -108,7 +109,7 @@ export class AppPage {
       annotationElement.dispatchEvent(mouseDownEvent);
       annotationElement.dispatchEvent(mouseMoveEvent);
       annotationElement.dispatchEvent(mouseUpEvent);
-    });
+    },xAxis,yAxis);
```

Comment:
why are you returning the values here, are they needed?

Reply:
yes thats how its been consumed within the block